include/uapi/linux/pps.h

```diff
@@ -26,7 +26,6 @@
 #include <linux/types.h>
 
 #define PPS_VERSION		"5.3.6"
-#define PPS_MAX_SOURCES		16		/* should be enough... */
```

Comment:
I fear that removing this constant might fun afoul of the "we don't break userspace" rule, but I don't see any other attempts elsewhere to plumb a kernel-config-defined constant into the uapi headers in order to maintain it accurately.

At the same time, uapi-header-defined constant limits are kind of bad in a world where we have sysfs and something can just enumerate /sys/class/pps, and at least in a github code search I see no other uses of that constant that are not coming from the many copies of the Linux kernel tree.

I'm okay with this in our tree but I know it's likely to become an issue when it goes upstream...

Reply:
Any suggestions here on how to avoid that?  I might continue to refine before sending upstream, but may also make that a separate change...

Reply:
The simplest might be to just change the comment from `/* should be enough... */` to `/* deprecated, use sysfs instead */`.

Reply:
fixed in v2.